JavaScript/Topics/5.Syntax/1.Data_Types/README.md

```diff
@@ -100,4 +106,29 @@ let person = {
     age:50, 
     eyeColor:"blue"};
 ```
+## 2. Arrays
+Arrays are collection of values of similar data type.
```

Comment:
In javascript, arrays can be a collection of different data types

Reply:
Sir, I've made the changes now review it.